bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -58,6 +58,7 @@ PreProcessorRecorder preProcessorRecorder;
 std::shared_ptr<concord::performance::PerformanceManager> sdm = make_shared<concord::performance::PerformanceManager>();
 
 uint64_t reqRetryId = 20;
+ViewNum viewNum = 1;
```

Comment:
As you don't modify this number anywhere => define the const variable in the beginning of the file.

Reply:
Already done at preprocessor_test.cpp.